database.md

```diff
@@ -0,0 +1,89 @@
+lastBlock | The last seen block.
+
+
+## Table `jobs`
+
+Stores information about jobs that the transcoder is assigned to.
+
+Column | Type | Description
+--- | --- | ---
+id |  INTEGER PRIMARY KEY| The ID of the on-chain job
+recordedAt | STRING DEFAULT CURRENT_TIMESTAMP | Time this row was inserted, *after* the job transaction has been confirmed on chain.
+streamID | STRING | Stream identifier, composed of `<NodeID><VideoID><Rendition>`.
+segmentPrice | INTEGER | The cost of transcoding each segment, in wei.
+transcodeOptions | STRING | [Encoded version of the rendition strings](https://github.com/livepeer/go-livepeer/blob/fac0393950e821f16555a17eb460b8abe74f6ada/common/videoprofile_ids.go#L5-L15).
+broadcaster | STRING | Broadaster's Eth address.
+transcoder | STRING | Transcoder's Eth address. Should be your node's address.
+startBlock | INTEGER | Block height from when the job was submitted on-chain.
+endBlock | INTEGER | Block height at which the job expires.
+stopReason | STRING | Reason for stopping a job on the transcoder. This may be before `endBlock` is reached. **Deprecated**
+stoppedAt | STRING | Time which the job was stopped on the transcoder. **Deprecated**
+
+## Table `claims`
+
+Stores and tracks claims as they go through the submission process.
+
+Column | Type | Description
+--- | --- | ---
+id | INTEGER | The claim ID. Starts at zero for every job.
+jobID | INTEGER | The ID of the job to claim
+claimRoot | STRING | Merkle root of the claim
+claimBlock | INTEGER | Block the claim tx was submitted at
+claimedAt | STRING DEFAULT CURRENT_TIMESTAMP | Time of submission
+updatedAt | STRING DEFAULT CURRENT_TIMESTAMP | Time of last update
+status  | STRING DEFAULT 'Created' | Claim status. May indicate an incomplete claim or error if the status is not `Complete`.
+&nbsp; | PRIMARY KEY(id, jobID) | Composite PK to ensure we don't try to submit a claim several times.
+&nbsp; | FOREIGN KEY(jobID) REFERENCES jobs(id) | FK constraints are not enforced at the moment
+
+## Table `receipts`
+
+Tracks segments as they go through the transcoding process.
+
+Column | Type | Description
+--- | --- | ---
+jobID |  INTEGER NOT NULL | The job this receipt belongs to.
+claimID | INTEGER | The claim in which this receipt was submitted at. May be NULL if a claim was not yet submitted.
+seqNo | INTEGER NOT NULL  | Sequence  number of this receipt.
+bcastFile | STRING | Location on disk of the broadcast file. After verification succeeds, this file may not exist anymore.
+bcastHash | STRING | Hash of the source segment from the broadcaster.
+bcastSig | STRING | Broadcaster's signature verifying this segment, approximately `Sign(broadcasterKey, Hash(streamID \| seqNo \| bcastHash))` 
+transcodedHash | STRING | Hash of the concatenation of all transcoded outputs for this segment,
+ `Hash(Hash(Transcode(Seg, Profile1)) \| ... \| Hash(Transcode(Seg, ProfileN)))` 
```

Comment:
Just a minor note on how this is rendered - looks like this line ended up in its own row

Reply:
Fixed 👍